website/general_issues.qmd

```diff
@@ -69,4 +69,13 @@ All wrappers must still be placed inside the `\examples{}` tag.
 
 :::
 
+# Package Size
+
+CRAN package tarballs should be under 5 Mb, with very few exceptions (and these will trigger flags for every future submission, so it's best to minimize the package size). Large datasets can be hosted externally, such as on GitHub, and downloaded when necessary. Consider placing them in a separate data package that requires less frequent updates.
```

Comment:
The [CRAN policy](https://cran.r-project.org/web/packages/policies.html) doesn't match this paragraph:

> As a general rule, neither data nor documentation should exceed 5MB (which covers several books). A CRAN package is not an appropriate way to distribute course notes, and authors will be asked to trim their documentation to a maximum of 5MB. 

I've heard of CRAN packages with several MB on their inst/ folder that were accepted on CRAN.

Reply:
I'm going to defer to @Baltmann01 on this difference from CRAN policy. Perhaps we should bold **very few exceptions** wording? I also think that the general issues section of the cookbook is more along the goal of best practices and CRAN volunteer-specific recommendations.

Reply:
I suggest rephrasing the recipe a bit to something like: 
```suggestion
Ideally, CRAN packages should be under 5 Mb. There are *very few exceptions* mostly for a larger inst/ folder (and these will trigger flags for every future submission). They must be approved by a CRAN team member and since every submitted version is saved on CRAN, they are very hesitant.
So it's best to minimize the package size. Large datasets can be hosted externally, such as on GitHub, and downloaded when necessary. Consider placing them in a separate data package that requires less frequent updates.
```
That is more along the lines with the policy. The inst/ folder can sometimes be larger but ideally not more than 10Mb max. Generally, if a package is larger than 5Mb I will ask the maintainer to reduce the size. If that is really not feasible and would influence package functionality, we'll probably make an exception (to some extend).

Reply:
With tools::CRAN_current_db one can check this: rcdklibs_2.9.tar.gz is ~19Mb... But yes this is only a minority 0.6% that go beyond the 5Mb.
Important note: Not every submission version is saved on CRAN archive, only those accepted...